### PR TITLE
DTSPO-1187 Updated GitHub user onboarding process

### PR DESCRIPTION
**Pull request by mokainos**
https://tools.hmcts.net/jira/browse/DTSPO-1187

### CODE REVIEW

File: source/onboarding/team/index.html.md.erb [hunk 1]
```diff
@@ -36,9 +36,18 @@ Make sure you pick the Jenkins \[app] user:
...
-if in doubt ask your delivery manager).
+Steps to add a user in GitHub
+
+1. Create a branch in [github-management](https://github.com/hmcts/github-management)
+2. Add your team member to [github-users.yaml](https://github.com/hmcts/github-management/blob/main/users/github-users.yaml) with their GitHub username, HMCTS email account and agree_to_acceptable_use_policy set to true. 
+```yaml
+- github_username: 3xample
+  email_account: "example.user@HMCTS.NET"
+  agree_to_acceptable_use_policy: true
+```
```
Comment (timja) [inline source/onboarding/team/index.html.md.erb:49]: this doesn't belong here, specific documentation belongs in the repository

```suggestion
```

File: source/onboarding/person/index.html.md.erb [hunk 2]
```diff
@@ -24,7 +24,15 @@ Join the following
 Where `username` is the part of your email before @ (usually firstname.lastname like john.smith)
 These groups will give you access to the VPN, the [Azure Portal](https://portal.azure.com/) and logs.
 
-- Github - join the appropriate team group
+- GitHub Access - submit pull request to [github-management](https://github.com/hmcts/github-management) repository
```
Comment (timja) [inline source/onboarding/person/index.html.md.erb:27]: I assume you didn't check this locally, formatting is very broken

image

Read the [acceptable use policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance), if you agree to follow it then ask someone in your team who has access to add you in the [GitHub management](https://github.com/hmcts/github-management) repository.

File: source/onboarding/person/index.html.md.erb [hunk 3]
```diff
@@ -24,7 +24,15 @@ Join the following
...
+```yaml
+- github_username: 3xample
+  email_account: "example.user@HMCTS.NET"
+  agree_to_acceptable_use_policy: true
+```
+  Read [Acceptable Use Policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance) and request your PMO[Delivery Manager] to submit the PR.
```
Comment (timja) [inline source/onboarding/person/index.html.md.erb:33]: PMO isn't delivery manager and neither are involved in this process

**Review by timja — CHANGES_REQUESTED** [review]

File: source/onboarding/person/index.html.md.erb (hunk 2 above)
Comment (mokainos) [reply source/onboarding/person/index.html.md.erb:27]: Modified

File: source/onboarding/team/index.html.md.erb [hunk 4]
```diff
@@ -36,21 +36,12 @@ Make sure you pick the Jenkins \[app] user:
 ## Github config
 
 ### Access
-Once you've received your `hmcts.net` email account and `Confluence` access, you can request GitHub access through your PMO (probably who you showed your documents to on your first day, if in doubt ask your delivery manager) after reading mandatory [Acceptable Use Policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance).
+Once you've received your `hmcts.net` email account and `Confluence` access, you can request GitHub access in one of the two ways.
```
Comment (timja) [inline source/onboarding/team/index.html.md.erb:39]:
```suggestion
Once you've received your `hmcts.net` email account and `Confluence` access, you can request GitHub access in one of two ways.
```

File: source/onboarding/team/index.html.md.erb [hunk 5]
```diff
@@ -36,21 +36,12 @@ Make sure you pick the Jenkins \[app] user:
...
 
-Steps to add a user in GitHub
+1. If you're the first person in your team you should raise a help request in #platops-help, asking for github access, you need to provide your GitHub username and say that you agree to [Acceptable Use Policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance).
```
Comment (timja) [inline source/onboarding/team/index.html.md.erb:41]:
```suggestion
1. If you're the first person in your team you should raise a help request in [#platops-help](https://hmcts-reform.slack.com/archives/C8SR5CAMU), asking for GitHub access, you need to provide your GitHub username and say that you agree to the [Acceptable Use Policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance).
```

File: source/onboarding/team/index.html.md.erb [hunk 6]
```diff
@@ -36,21 +36,12 @@ Make sure you pick the Jenkins \[app] user:
...
+2. If someone in your team already has access then ask them to follow the instructions at [github-management](https://github.com/hmcts/github-management) repository to add you.
```
Comment (timja) [inline source/onboarding/team/index.html.md.erb:42]:
```suggestion
2. If someone in your team already has access then ask them to follow the instructions in the [GitHub management](https://github.com/hmcts/github-management) repository to add you.
```

File: source/onboarding/team/index.html.md.erb [hunk 7]
```diff
@@ -36,12 +36,12 @@ Make sure you pick the Jenkins \[app] user:
...
+Once you've received your `hmcts.net` email account and `Confluence` access, you can request GitHub access in one of the two ways.
 
-Request GitHub access through your PMO, (probably who you showed your documents to on your first day, 
-if in doubt ask your delivery manager).
+1. If you're the first person in your team you should raise a help request in #platops-help, asking for github access, you need to provide your GitHub username and say that you agree to [Acceptable Use Policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance).
+2. If someone in your team already has access then ask them to follow the instructions at [github-management](https://github.com/hmcts/github-management) repository to add you.
 
-You should receive an email inviting you to the GitHub organisation, visit [https://github.com/hmcts](https://github.com/hmcts)
-to accept the invite.
+You should receive an email inviting you to the GitHub organisation, visit [https://github.com/hmcts](https://github.com/hmcts) to accept the invite.
```
Comment (timja) [inline source/onboarding/team/index.html.md.erb:44]: In the above lines I would probably swap 1 and 2 around, self service is preferred.

File: source/onboarding/person/index.html.md.erb [hunk 8]
```diff
@@ -24,7 +24,15 @@ Join the following
 Where `username` is the part of your email before @ (usually firstname.lastname like john.smith)
 These groups will give you access to the VPN, the [Azure Portal](https://portal.azure.com/) and logs.
 
-- Github - join the appropriate team group
+- GitHub Access - Read the [acceptable use policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance), if you agree to follow it then ask someone in your team who has access to add you in the [GitHub management](https://github.com/hmcts/github-management) repository by submitting a PR.
```
Comment (timja) [inline source/onboarding/person/index.html.md.erb:27]:
```suggestion
- GitHub Access - Read the [acceptable use policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance), if you agree to follow it then ask someone in your team who has access to add you in the [GitHub management](https://github.com/hmcts/github-management) repository by submitting a pull request.
```

File: source/onboarding/person/index.html.md.erb [hunk 9]
```diff
@@ -24,7 +24,15 @@ Join the following
...
+
+```yaml
+- github_username: 3xample
+  email_account: "example.user@HMCTS.NET"
+  agree_to_acceptable_use_policy: true
+```
+
```
Comment (timja) [inline source/onboarding/person/index.html.md.erb:34]: documentation is in the repo for this and it's very straightforward
```suggestion
```

File: source/onboarding/person/index.html.md.erb [hunk 10]
```diff
@@ -24,7 +24,15 @@ Join the following
...
+- GitHub Teams - join the appropriate team group
```
Comment (timja) [inline source/onboarding/person/index.html.md.erb:35]: more information is required here, 

what about?
```suggestion
- GitHub Teams - ask your Technical Lead /or a senior member of the team to add you to your GitHub team group.
```

**Review by timja — CHANGES_REQUESTED** [review]
better but needs a few tweaks

File: source/onboarding/team/index.html.md.erb [hunk 11]
```diff
@@ -36,12 +36,12 @@ Make sure you pick the Jenkins \[app] user:
 ## Github config
 
 ### Access
+Once you've received your `hmcts.net` email account and `Confluence` access, you can request GitHub access in one of two ways.
```
Comment (timja) [inline source/onboarding/team/index.html.md.erb:39]:
```suggestion
Once you've received your `hmcts.net` email account and `Confluence` access, you can request GitHub access in one of two ways:
```

File: source/onboarding/person/index.html.md.erb [hunk 12]
```diff
@@ -24,7 +24,8 @@ Join the following
 Where `username` is the part of your email before @ (usually firstname.lastname like john.smith)
 These groups will give you access to the VPN, the [Azure Portal](https://portal.azure.com/) and logs.
 
-- Github - join the appropriate team group
+- GitHub Access - Read the [acceptable use policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance), if you agree to follow it then ask someone in your team who has access to add you in the [GitHub management](https://github.com/hmcts/github-management) repository by submitting a pull request.
+- GitHub Teams - ask your Technical Lead /or a senior member of the team to add you to your GitHub team group.
```
Comment (timja) [inline source/onboarding/person/index.html.md.erb:28]:
```suggestion
- GitHub Teams - ask your Technical Lead or a senior member of the team to add you to your GitHub team group.
```

**Review by timja — APPROVED** [review]

**Review by adusumillipraveen — APPROVED** [review]

**Review by cakeben — APPROVED** [review]

**Review by njrich28 — APPROVED** [review]
LGTM

**Pull request merged**